src/com/biocollapse/controller/SimulationController.java

```diff
@@ -87,4 +90,14 @@ private void updateSimulation() {
         }
         infectionService.updateHumansStatus(humans, tick);
     }
+
+    private boolean isSimulationComplete(LiveStatistics currenStatistics){
+        boolean result = false;
+        
+        // Check for: Max Days, No one infected, all dead
+        if (currenStatistics.getInfected() == 0 || currenStatistics.getAlive() == 0 || tick > (250 * 14)) {
```

Comment:
It would be great if you could use the `TICKS_PER_DAY` variable in the config (which idea i specified in another comment) and add another variable to set the number of days for the simulation. Just to avoid magic numbers :)

Reply:
Yess... this kinda happened in #31. We should resolve this before doing the work twice :')

```
    private final static int SIMULATION_ONE_DAY_TICKS = 250;
    private final static int SIMULATION_MAX_DAYS = 250;
```